arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+        },
+        "eventName": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the event name.This value should not be confused with OperationName.For practical purposes, OperationName might be more appealing to end users."
+        },
+        "category": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the event category."
+        },
+        "httpRequest": {
+          "$ref": "#/definitions/HttpRequestInfo",
+          "description": "Gets or sets the HTTP request info.The client IP address of the user who initiated the event is captured as part of the HTTP request info."
+        },
+        "level": {
+          "type": "string",
+          "description": "Gets or sets the event level",
+          "enum": [
+            "Critical",
+            "Error",
+            "Warning",
+            "Informational",
+            "Verbose"
+          ],
+          "x-ms-enum": {
+            "name": "EventLevel",
+            "modelAsString": false
+          }
+        },
+        "resourceGroupName": {
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790546.aspx"
+          },
+          "type": "string",
+          "description": "Gets or sets the resource group name."
+        },
+        "resourceProviderName": {
+          "$ref": "#/definitions/LocalizableString",
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790572.aspx"
+          },
+          "description": "Gets or sets the resource provider name."
+        },
+        "resourceId": {
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790569.aspx"
+          },
+          "type": "string",
+          "description": "Gets or sets the resource uri"
+        },
+        "resourceType": {
+          "$ref": "#/definitions/LocalizableString",
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790569.aspx"
+          },
+          "description": "Gets or sets the resource type"
+        },
+        "operationId": {
+          "type": "string",
+          "description": "Gets or sets the operation idThis value should not be confused with EventName."
+        },
+        "operationName": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the operation name."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets the property bag"
+        },
+        "status": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the event status.Some typical values are: Started, Succeeded, Failed"
```

Comment:
Please update the description
Is a string describing the status of the operation. Here are some of the common values for status:
Started
In Progress
Succeeded
Failed
Active
Resolved